store/sqlstore/team_store.go

```diff
@@ -155,10 +150,9 @@ func (db teamMemberWithSchemeRolesList) ToModel() []*model.TeamMember {
 	return tms
 }
 
-func NewSqlTeamStore(sqlStore SqlStore, metrics einterfaces.MetricsInterface) store.TeamStore {
+func NewSqlTeamStore(sqlStore SqlStore) store.TeamStore {
```

Comment:
Why are we changing this function [prototype](https://en.wikipedia.org/wiki/Function_prototype)?

Reply:
I changed it because the metrics is not used anymore for this case, it was used only to report about the cache but it was removed.

I this is wrong I would revert this change ;)

Reply:
It's not wrong but it's not a backward compatible change. I would probably treat it the same way as the `allowFromCache` param.